metricbeat/module/docker/container/_meta/fields.yml

```diff
@@ -40,3 +40,24 @@
           type: long
           description: >
              Size of the files that have been created or changed since creation.
+    - name: health
```

Comment:
For the field naming, please check our naming conventions: https://www.elastic.co/guide/en/beats/libbeat/5.1/event-conventions.html